omnibus/config/templates/datadog-agent/systemd.network.service.erb

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description="Datadog Network Tracer"
+After=network.target datadog-agent.service
+BindsTo=datadog-agent.service
+StartLimitIntervalSec=10
+StartLimitBurst=5
+
+[Service]
+Type=simple
+PIDFile=<%= install_dir %>/run/network-tracer.pid
+Restart=on-failure
+ExecStart=<%= install_dir %>/embedded/bin/network-tracer --config=<%= etc_dir %>/datadog.yaml --pid=<%= install_dir %>/run/network-tracer.pid
```

Comment:
there's no `User` specified here, does that mean that the process runs as `root`?

Reply:
thanks for the link 👍 

Even if the network-tracer shuts down almost immediately by default, it's still running as `root` to do so, so we may want to avoid starting the service by default with the `datadog-agent` service (all of the Agent processes are supposed to run as `dd-agent`). What do you think?

Also, could you get this reviewed by the ProdSec team please? :)

Reply:
I'm all up for avoiding starting this service by default. I'm not sure what the workflow to enable it would look like though.

Yes, can absolutely request to get this reviewed by prodsec

Reply:
if the network tracer is not started automatically by the main Agent service, users would have to start the `datadog-agent-network` service separately from the `datadog-agent` service

Reply:
+1 I think this is a good approach while we alpha/beta. The only downside I can think of is that once we want to work with real customers (before making it part of the main service), they will need to make this work for their config management tool. But I think it will be a limited set and we can work with them to make it happen.